mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java

```diff
@@ -91,6 +96,7 @@
 	private Connection connection;
 	private Session session;
 	private Destination destination;
+	private Integer textType = 0;
```

Comment:
create it as a local variable not a global variable

Reply:
Done changes